### DISCUSSION TITLE
local scan for subdomains like on the online platform

### DISCUSSION THREAD

**Discussion by ImVantexHD** [projectdiscovery/nuclei#5654] [Q&A]
hello, im using https://cloud.projectdiscovery.io/assets for scanning subdomains which is very effective but has limits (You can Discover up to 10 domains per month).
i want to know what is the equivalent command to perform a local scan using projectdiscovery tools, if possible keeping the same parameters as they are on site.

i tried 'subfinder -all' but it doesn't seem to identify all the subdomains that the online platform uncovers.

**Comment by dogancanbakir** [comment]
Replicating things in a non-deterministic environment like the web can be quite challenging. In the subfinder, we're relying on the sources. Usually, all you get is what they provide, except some are excluded via post-processing. Even with all the [sources properly configured](https://docs.projectdiscovery.io/tools/subfinder/install#post-install-configuration), replication may not be achievable, as each running instance can vary significantly from the others.
